src/olympia/stats/templatetags/jinja_helpers.py

```diff
@@ -15,6 +15,7 @@ def report_menu(context, request, report, obj):
     if isinstance(obj, Addon):
         tpl = loader.get_template('stats/addon_report_menu.html')
         ctx = {
-            'addon': obj,
+            'has_listed_versions': obj.has_listed_versions(include_deleted=True),
+            'slug_or_id': obj.id if obj.is_deleted else obj.slug,
```

Comment:
Why not just use `obj.slug` if it exists, otherwise use `obj.id`, rather than specifically checking `obj.is_deleted`? Or must we use `obj.id` if the add-on has been deleted?

Reply:
we use `obj.id` only when the add-on is deleted, indeed.